designsafe/apps/auth/views.py

```diff
@@ -161,3 +157,21 @@ def tapis_oauth_callback(request):
         return HttpResponseRedirect(next_uri)
 
     return HttpResponseRedirect(settings.LOGIN_REDIRECT_URL)
+
+
+def logged_out(request):
+    """Render logged out page upon logout"""
+    return render(request, "designsafe/apps/auth/logged_out.html")
+
+
+class LogoutView(DjangoLogoutView):
+    def dispatch(self, request):
+        token = request.user.tapis_oauth.access_token
+        logger.info('The user %s is attempting to logout via Tapis with token "%s"' % (request.user.username, token[:8].ljust(len(token), '-')))
+
+        logout_endpoint = f"{settings.TAPIS_TENANT_BASEURL}/v3/oauth2/logout?redirect_url=https://{request.get_host()}/{settings.LOGOUT_REDIRECT_URL}"
+
+        logout(request)
```

Comment:
Should we be logging out after a successful logout response? i.e. after `response = HttpResponseRedirect(logout_endpoint)`